src/components/settings/PresetButtons.svelte

```diff
@@ -0,0 +1,56 @@
+<script>
+  import {
+    captionDuration,
+    captionFontSize,
+    videoSideSetting,
+    chatZoom,
+    livetlFontSize,
+    textDirection,
+    chatSplit,
+    autoVertical,
+    autoLaunchMode,
+    activePreset,
+    presets
+  } from '../../js/store';
+  import { importStores } from '../../js/storage';
+  import Button from 'smelte/src/components/Button';
+
+  $: active = $activePreset;
+  $: activePreset.set(active);
+
+  /**
+  * @param {number} number
+  */
```

Comment:
We started using typescript instead of jsdoc, can you change this to a ts hint and make the script tag a `<script lang="ts">`?